src/proof_system/preprocess.rs

```diff
@@ -121,168 +121,168 @@ impl TurboComposer {
         let (_, selectors, preprocessed_table, domain) =
             self.preprocess_shared(commit_key, transcript)?;
 
-        let domain_4n = EvaluationDomain::new(4 * domain.size())?;
-        let q_m_eval_4n = Evaluations::from_vec_and_domain(
-            domain_4n.coset_fft(&selectors.q_m),
-            domain_4n,
+        let domain_8n = EvaluationDomain::new(8 * domain.size())?;
```

Comment:
Maybe add a comment that explains the `8`

Reply:
I'm also not sure how this is related to the blinding scalars. Can we document somewhere (issue?) why we need to do this in order to add blinding scalars?

Reply:
we are adding these values, which increases the size of the polynomial by 2 degrees. This means that the polynomial used before no longer fits into `4n`, so, the next power of 2 is `8n`.

Reply:
As side note, we shouldn't add comments that depends on historical context to the static code. Instead, we need to put self-resolving complete information.

Something like "8n domain, being 4n *add here some PLONK context*, xn for the blinders, and the remainder to fit the next power of 2`